pkgs/top-level/all-packages.nix

```
@@ -3693,6 +3693,8 @@ in
 
   gitaly = callPackage ../applications/version-management/gitlab/gitaly { };
 
+  gitqlient = qt5.callPackage ../applications/version-management/gitqlient { };
```

It seems it is much more common to use 

```suggestion
  gitqlient = libsForQt5.callPackage ../applications/version-management/gitqlient { };
```